less/index.less

```diff
@@ -1,59 +1,238 @@
-// Is this working?
-/* http://meyerweb.com/eric/tools/css/reset/ 
-   v2.0 | 20110126
-   License: none (public domain)
-*/
-
-html, body, div, span, applet, object, iframe,
-h1, h2, h3, h4, h5, h6, p, blockquote, pre,
-a, abbr, acronym, address, big, cite, code,
-del, dfn, em, img, ins, kbd, q, s, samp,
-small, strike, strong, sub, sup, tt, var,
-b, u, i, center,
-dl, dt, dd, ol, ul, li,
-fieldset, form, label, legend,
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, canvas, details, embed, 
-figure, figcaption, footer, header, hgroup, 
-menu, nav, output, ruby, section, summary,
-time, mark, audio, video {
-	margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
+@cool-color:white;
+@font-content: 'Play', serif;
+@font-header: 'ZCOOL XiaoWei', serif;
+@display-flex:flex;
+
+html, body {
+    font-size: 62.5%;  
+    // font-family: 'ZCOOL XiaoWei', serif;
+    font-family: 'Play', sans-serif;
 }
-/* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+h1, h2, h3, {
+    font-family: 'ZCOOL XiaoWei', serif;
+    letter-spacing: 1px;
+	margin-bottom: 0.5rem;
+	
+	h1{
+		font-size: 10rem;
+	}
+
+.container {
+    max-width:1200px;
+    margin: auto;
+	font-size:1.4rem;
+	justify-content: space-evenly;
 }
-body {
-	line-height: 1;
+button.btn {
+	margin: 20px 0 20px;
+	padding: 1.2em 2.8em;
+	height: 300px;
+	width: 30px;
+	border: 2px solid red;
+
+	&:hover,
+	&:focus {
+	  color: @cool-color;
+	  outline: 0;
+	}
+  .third {
+	border-color: blue;
+	color: @cool-color;
+	box-shadow: 0 0 40px 40px blue inset, 0 0 0 0 blue;
+	transition: all 150ms ease-in-out;
+	
+	&:hover {
+	  box-shadow: 0 0 10px 0 blue inset, 0 0 10px 4px blue;
+	}
+  }
 }
-ol, ul {
-	list-style: none;
+
+header nav {
+    margin: 2rem;
+    margin-top: 4rem; 
+	width: 100%;
+	float: left;
+    justify-content: space-evenly;
+	display: flex;
+
+	
+}
+div {
+    background-color: @cool-color;
+     color:grey;
+     padding: 25px;
+     margin: 10px auto;
+	 display: inline-block;
+	 justify-content: space-evenly;
```

Comment:
Still need to work a little more on formatting, there are whitespaces everywhere around the file. Try the `prettier` plugin for VS Code which can take of these for You whenever You save changes.